packages/web/package.json

```diff
@@ -28,18 +29,17 @@
     "sass": "^1.69.7",
     "socket.io-client": "^4.7.4",
     "three": "^0.162.0",
+    "uuid": "^10.0.0",
     "web-vitals": "^3.5.1"
   },
   "peerDependencies": {
     "react": "^18.0.0",
     "react-dom": "^18.0.0"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "dev": "nodemon start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "vite",
```

Comment:
Big change -> Using [vite](https://vitejs.dev/) instead of react-scripts. I did this because we have to include env variables in the frontend code, and vite makes this really easy (see [web/src/index.tsx](https://github.com/laubsauger/osc.link/pull/3/files#diff-35658c808555b4731fdaef6b64a8f90753819acf6aab96edb67ec88ff0ae3e5dR19)). 

Same dev experience, more up-to-date tooling.

Reply:
Still didnt get to use to dabble with vite at all so no clue and no real opinion.
if it's only about ENV vars, dotenv does that for you all day, no?

Reply:
Not only about env vars, but it was the impetus to do the update.  I like the way Vite handles them, and it was straightforward to do.

- react-scripts is no longer maintained, with multiple deprecation warnings appearing during npm install. The package hasn't been updated in over two years.
- Migrating to Vite was straightforward, is actively maintained and includes useful quality-of-life features like built-in [env var management](https://vitejs.dev/guide/env-and-mode.html).
- We don’t interact with Vite directly; it functions similarly to react-scripts in handling the build process via npm scripts.